src/styles/index.scss

```diff
@@ -1,3 +1,150 @@
-body {
+* {
+  box-sizing: border-box;
+}
+
+body,
+p,
+ul {
   margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+:root {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+$color__text--main: #060b35;
+$color__text--secondary: #616070;
+$color__text--accent: #00acdc;
+$color__background--accent: #00acdc;
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color__text--main;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $color__text--secondary;
+    margin-bottom: 16px;
+  }
+
+  &__rating,
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__rating {
+    margin-bottom: 24px;
+
+    color: #060b35;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+    color: $color__text--secondary;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__buy-button {
+    box-sizing: border-box;
+
+    height: 40px;
+    margin: 16px 0;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    text-decoration: none;
+    background-color: $color__text--accent;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    color: white;
+    text-transform: uppercase;
+
+    border: 1px solid $color__text--accent;
+    border-radius: 5px;
+
+    &:hover {
+      background-color: #fff;
+      color: $color__background--accent;
+      cursor: pointer;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+
+  list-style-type: none;
+
+  &__star {
+    margin-left: 4px;
+
+    height: 16px;
+    width: 16px;
+
+    background-image: url(/src/images/star.svg);
```

Comment:
According to the task, path to image should be relative to index.scss file - i.e., starting from '../images/' in this case. In your solution path is absolute. So please change it to relative format here and in line 148.